torchvision/prototype/models/depth/stereo/raft_stereo.py

```diff
@@ -448,6 +455,10 @@ def forward(self, image1: Tensor, image2: Tensor, num_iters: int = 12) -> List[T
         coords0 = make_coords_grid(batch_size, Hf, Wf).to(fmap1.device)
         coords1 = make_coords_grid(batch_size, Hf, Wf).to(fmap1.device)
 
+        # We use flow_init for cascade inference
+        if flow_init is not None:
+            coords1 = coords1 + flow_init
```

Comment:
This is the only change for which I don't have context, but I understand it's a new feature.

Reply:
When wanting to perform inference at a resolution significantly larger than that at which the model is trained, you can perform cascaded inference.

Cascaded inference first computes a flow at a low resolution, and uses that flow as a prior for the larger resolution image.